### PR TITLE
feat/AUT-1612/allow set QTI Identifier pattern from configuration

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-1612

Added support to update QTI Identifier pattern from configuration for Item and Test Authoring

**How to test:**
1. set in config new pattern `config/tao/client_lib_config_registry.conf.php`
```
        'taoQtiItem/qtiCreator/widgets/helpers/qtiIdentifier' => array(
            'qtiIdPattern' => '^[a-zA-Z_][a-zA-Z0-9_-]*$'
        )
```
2. check that in Item and Test Authoring QTI identifier for test, test-part, test-section and assessmentItemRef with dot is not valid (error message will stay the same)

### CODE REVIEW

File: model/qti/ServiceProvider/ItemIdentifierValidatorServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+class ItemIdentifierValidatorServiceProvider implements ContainerServiceProviderInterface
+{
+    public function __invoke(ContainerConfigurator $configurator): void
+    {
+        $services = $configurator->services();
+        $parameters = $configurator->parameters();
+
+        /**
+         * Please specify the QTI Identifier Validator Pattern that will be used to validate the item identifier
```
Comment (pnal) [inline model/qti/ServiceProvider/ItemIdentifierValidatorServiceProvider.php:39]: Do we have any mentions in documentation about that feature?

Comment (pnal) [reply model/qti/ServiceProvider/ItemIdentifierValidatorServiceProvider.php:39]: You introduced new configurable feature, so now anyone is able to add .env variable and change the behavior of the product, do you think it's fine to leave it to be found only by investigating the code?

**Review by pnal — APPROVED** [review]
BE approved.
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by Silvia-loza — APPROVED** [review]
Checked FE code for this task and tested in local (not code review for BE)
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not ` master`
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Pull request merged**